src/control/server/storage/scm_test.go

```diff
@@ -1,4 +1,48 @@
 //
+// (C) Copyright 2019 Intel Corporation.
```

Comment:
seems to have multiple copyright entries on this file

Reply:
Weird, not sure how that happened.

Reply:
fixed